packages/server/postgres/migrations/1676299334703_addSeasonalTemplates.ts

```diff
@@ -0,0 +1,213 @@
+    templateId: makeId(templateNames[0], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Easter Eggs',
+    description:
+      'What was hidden from sight or caught you by surprise? What did you learn or find out?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Seeds',
+    description:
+      'What new processes would we like to cultivate? Is there something that faded away that we should bring back?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Hope',
+    description: 'What did you hope for? What are you hoping for next?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 2
+  },
+  {
+    question: 'Chocolate',
+    description: 'Who deserves some kudos or a word of thanks?',
+    templateId: makeId(templateNames[1], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Wreaths',
+    description: 'What processes and practices are binding us together and making us stronger?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Schnapps',
+    description: 'What’s been hard to swallow, admit, or realize about our work?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Longest Day',
+    description: 'What feels like it took too long or was never-ending?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 2
+  },
+  {
+    question: 'Strawberry Cake',
+    description: 'Who deserves some kudos or a word of thanks?',
+    templateId: makeId(templateNames[2], 'template'),
+    sortOrder: 3
+  },
+  {
+    question: 'Hongbao 🧧',
+    description: 'Who deserves a red envelope with kudos and some words of appreciation?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 0
+  },
+  {
+    question: 'Couplets 🎊 ',
+    description: 'What important messages should we remember for guidance?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 1
+  },
+  {
+    question: 'Firecracker 🧨',
+    description: 'What caught us by surprise or gave us a fright?',
+    templateId: makeId(templateNames[3], 'template'),
+    sortOrder: 2
+  }
+]
+
+const templates = templateNames.map((templateName) => makeTemplate(templateName))
+const reflectPrompts = promptsInfo.map((promptInfo, idx) => makePrompt(promptInfo, idx))
+
+const connectRethinkDB = async () => {
+  const {hostname: host, port, pathname} = new URL(process.env.RETHINKDB_URL!)
+  await r.connectPool({
+    host,
+    port: parseInt(port, 10),
+    db: pathname.split('/')[1]
+  })
+  return r as any as ParabolR
+}
+
+export const up = async function () {
+  try {
+    await connectRethinkDB()
+    await Promise.all([
+      r.table('MeetingTemplate').insert(templates).run(),
+      r.table('ReflectPrompt').insert(reflectPrompts).run()
+    ])
+    await r.getPoolMaster()?.drain()
+  } catch (e) {
```

Comment:
-1 Any reason why we ignore errors here?

Reply:
That's the pattern we have in many of our other migrations, see [here](https://github.com/ParabolInc/parabol/blob/ec1cc40f177b02a0a5f37b8e7cd7b422b3f7961e/packages/server/database/migrations/20220602164119-fixRetroTemplates.ts) for example. Would you prefer to send the error to Sentry?

Reply:
Let's go with it, it will not break the app if we're missing some templates. Otherwise we should not ignore errors in migrations.